versions.tf

```diff
@@ -18,12 +24,12 @@ terraform {
 
     kubernetes = {
       source  = "hashicorp/kubernetes"
-      version = "2.0.1"
+      version = "2.11.0"
```

Comment:
Are we sure that this version is stable enough to upgrade to ? if not please do a round of tests just to be 100% sure